strings/tips.txt

```diff
@@ -273,11 +274,29 @@ Fun fact: reading this (https://beestation13.com/rules) may lead to an increase
+Whenever you have a corpse that's not on a stasis bed, jab them with an epi-pen, as they contain formaldehyde, which will halt organ decay.
+The tools in the surgical toolset arm are the second fastest surgical tools available, outsped only by alien tools.
+There are multiple ways to make surgery steps take less time. Operating tables will provide a speed boost compared to stasis beds (and alien operating tables even more so), surgery on sleeping or dead patients will be faster than awake patients, and various reagents such as honey, sterilizer spray, or alcohol can be applied after a surgery is started to provide a minor speed boost.
+You can use Cortex Imprint surgery, unlocked by Experimental Surgery tech, in order to render someone immune to mild brain traumas. This will not heal damage to the brain itself, however.
+Inacusiate will instantly heal all hearing damage whenever it is ingested, regardless of the amount ingested. A 0.1 unit pill will cure deafness just fine.
+The Mind Control nanite program, unlocked with Hazard Nanite Programs (which itself requires Alien Technology), completely bypasses mindshields, however it only lasts for 1 minute, and has a 3 minute cooldown before it can be triggered again.
+Hypnotic flashes will only induce a trance under certain conditions: the target must not be mindshielded, and they must either be asleep, hallucinating, suffering from reduced brain activity, or have low sanity.
+Advanced surgeries can only be started on an operating table. Don't try to start a revival surgery on a stasis bed, you won't see the option.
+Don't overuse bruise packs or ointment on people, as these contain styptic powder and silver sulfadiazine respectively, which are capable of overdosing someone and causing suffocation damage if overused!
+Wound tending surgery works perfectly fine on IPCs. Don't question it.
+The brain traumas associated with hypnotic flashes and obsessions can only be cured through surgery — nanites and viruses will completely ignore them!
+You can free someone from an abductor-induced brainwashing by replacing the fleshy gland in their chest with a proper heart, and then performing a lobotomy on them.
+Changeling transformation stings can be reversed using clonexadone — it does not require a specific unit amount, and instead will be reversed after clonexadone has been in the victim's body for about a minute or so.
+When performing wound tending or blood filtering surgery, you can hold a health analyzer in your other hand in order to see how much damage is remaining on your patient.
+Oozelings can hug flaming people in order to help extinguish them.
+Trying to talk things out can often result in a better outcome than resorting to violence.
+Ghouls and voiceless dead (which can be identified as having a "blank, catatonic-like stare" when examined) will revert back to normal upon dying, restoring their appearance and freeing them from their loyalty to the heretic that converted them.
```

Comment:
This tip gives confusing information as disconnected players have the same message. This encourages players to kill anyone with that message

Reply:
Hm, fair. I was trying to word it literally, but fair point:
```dm
/datum/status_effect/ghoul
	id = "ghoul"
	status_type = STATUS_EFFECT_UNIQUE
	duration = -1
	examine_text = "<span class='warning'>SUBJECTPRONOUN has a blank, catatonic like stare.</span>"
```